src/ui/view/vault.rs

```diff
@@ -87,7 +89,20 @@ impl VaultModal {
                                                         .push(badge::tx_deposit())
                                                         .push(
                                                             Column::new()
-                                                                .push(text::small(&vlt.txid))
+                                                                .push(
+                                                                    Row::new()
+                                                                        .push(text::small(
+                                                                            &vlt.txid,
+                                                                        ))
+                                                                        .push(button::clipboard(
+                                                                            &mut self.copy_button,
+                                                                            Message::Clipboard(
+                                                                                vlt.txid
+                                                                                    .to_string(),
+                                                                            ),
+                                                                        ))
+                                                                        .align_items(Align::Center),
+                                                                )
```

Comment:
Ok; so the intent is to be able to copy the deposit txid.

Reply:
I use this as an example, but this button will be added where our customers will ask us for better user experience